src/SFA.DAS.PR.Data/EntityConfiguration/AccountConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SFA.DAS.PR.Domain.Entities;
+using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.PR.Data.EntityConfiguration;
+
+[ExcludeFromCodeCoverage]
+public class AccountConfiguration : IEntityTypeConfiguration<Account>
+{
+    public void Configure(EntityTypeBuilder<Account> builder)
+    {
+        builder.HasKey(p => p.Id);
+
+        builder
+            .HasMany(a => a.AccountProviders)
```

Comment:
I think I may asked this before, but I cannot find my comment, do we need to explicitly state the relations as EF works by convention and most of the entities are conventionally correct.